{"schemaVersion":{"minor":3,"major":0,"patch":0},"sections":[],"abstract":[{"text":"Represents an error with a specific error code and message that the Nfinity Cloud might return, but not from network call.","type":"text"}],"hierarchy":{"paths":[["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudError(errorCode:errorMessage:)"},"primaryContentSections":[{"declarations":[{"platforms":["Mac Catalyst"],"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"cloudError"},{"kind":"text","text":"("},{"text":"errorCode","kind":"externalParam"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?, "},{"kind":"externalParam","text":"errorMessage"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?)"}]}],"kind":"declarations"},{"parameters":[{"name":"errorCode","content":[{"inlineContent":[{"text":"An optional error code as a string, that helps identify the type of error.","type":"text"}],"type":"paragraph"}]},{"name":"errorMessage","content":[{"inlineContent":[{"text":"An optional human-readable message providing more details about the error.","type":"text"}],"type":"paragraph"}]}],"kind":"parameters"}],"variants":[{"paths":["\/documentation\/passwordandnfinityframework\/nfinityclouderror\/clouderror(errorcode:errormessage:)"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"cloudError"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"errorCode"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?, ","kind":"text"},{"kind":"externalParam","text":"errorMessage"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?)","kind":"text"}],"modules":[{"name":"PasswordAndNfinityFramework"}],"externalID":"s:27PasswordAndNfinityFramework0C10CloudErrorO05cloudF0yACSSSg_AEtcACmF","symbolKind":"case","title":"NfinityCloudError.cloudError(errorCode:errorMessage:)","roleHeading":"Case","role":"symbol"},"kind":"symbol","references":{"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/cloudError(errorCode:errorMessage:)":{"url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/clouderror(errorcode:errormessage:)","abstract":[{"text":"Represents an error with a specific error code and message that the Nfinity Cloud might return, but not from network call.","type":"text"}],"role":"symbol","type":"topic","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"cloudError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"errorCode","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":"?, "},{"text":"errorMessage","kind":"externalParam"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":"?)"}],"kind":"symbol","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/cloudError(errorCode:errorMessage:)","title":"NfinityCloudError.cloudError(errorCode:errorMessage:)"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework":{"role":"collection","title":"PasswordAndNfinityFramework","type":"topic","url":"\/documentation\/passwordandnfinityframework","kind":"symbol","abstract":[],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError":{"abstract":[{"text":"Represents an enum hierarchy for errors related to the NfinityÂ Cloud.","type":"text"}],"role":"symbol","kind":"symbol","title":"NfinityCloudError","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"NfinityCloudError","kind":"identifier"}],"type":"topic","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError","navigatorTitle":[{"text":"NfinityCloudError","kind":"identifier"}]}}}